client/authorization/components/signInDialog.js

```diff
@@ -4,22 +4,60 @@ import {connect} from 'react-redux';
 import * as actions from '../actions';
 import Popup from './popup';
 import SignInForm from './signInForm';
+import SignUpForm from './signUpForm';
 import classNames from './scss/button.module.scss';
+import TabBar from '../../components/tabBar';
+import classNames1 from './scss/signInForm.module.scss'
 
 class SignInDialog extends Component {
+    constructor(props){
+        super(props);
+        this.state = {selectedId: 1};
+    }
+
+    handleChanged = (tab) => {
+        const {showForm} = this.props;
+        this.setState({selectedId: tab});
+        showForm();
+    };
+
     render() {
-        const {isShown, showForm} = this.props;
+        const {isShown} = this.props;
         const {appbar__button} = classNames;
 
+        let tabs=[
+            {
+                id: 1,
+                text: "Sign in"
+            },
+            {
+                id: 2,
+                text: "Sign up"
+            }
+        ];
+
         return (
             <React.Fragment>
-                <button onClick={showForm} className={appbar__button}>
+                <button onClick={() => {this.handleChanged(1)}} className={appbar__button}>
```

Comment:
inline function. Plus make a constant for tab so you don't need to use magic numbers